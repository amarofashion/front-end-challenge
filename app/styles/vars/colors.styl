main = #fff
inverted = #262626
text = #333
text-red = #ce584e
text-gray = #999
light-white = #f8f8f8