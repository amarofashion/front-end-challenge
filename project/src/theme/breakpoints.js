export default {
  all: 0,
  mobile: 320,
  tablet: 768,
  desktop: 1200,
};
