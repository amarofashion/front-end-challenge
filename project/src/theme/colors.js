export default {
  primary: '#E56A54',
  secondary: '#92BFB1',
  dark: '#000',
  light: '#fff',
  gray: '#eee',
};
