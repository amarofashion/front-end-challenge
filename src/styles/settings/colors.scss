$color-white: #fff;

$color-black: #000;
$color-black-2: #212529;
$color-black-3: #333333;

$color-pink: #f8c1b8;

$color-red: #cf3838;

$color-gray: #ccc;
$color-gray-1: #999999;