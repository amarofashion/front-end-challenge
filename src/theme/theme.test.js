import colors from './colors';
import breakpoints from './breakpoints';

describe('theme', () => {
  it('should return correctly', () => {
    expect(colors).toMatchSnapshot({});
    expect(breakpoints).toMatchSnapshot({});
  });
});
