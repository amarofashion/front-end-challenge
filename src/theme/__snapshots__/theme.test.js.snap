// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme should return correctly 1`] = `
Object {
  "dark": "#000",
  "gray": "#eee",
  "light": "#fff",
  "primary": "#E56A54",
  "secondary": "#92BFB1",
}
`;

exports[`theme should return correctly 2`] = `
Object {
  "all": 0,
  "desktop": 1200,
  "mobile": 320,
  "tablet": 768,
}
`;
